scripts/install_chrome_on_travis_test.py

```diff
@@ -0,0 +1,91 @@
+# coding: utf-8
```

Comment:
Hmm, it looks like this was removed in a previous file, should we keep this or remove it?

Reply:
I think it's OK to leave it; it does no harm.